framework/web/JsonResponseFormatter.php

```diff
@@ -38,6 +38,11 @@
 class JsonResponseFormatter extends Component implements ResponseFormatterInterface
 {
     /**
+     * @var string|null the MIME type used in the `Content-Type` header of the response.
+     * When equals `null` default MIME type will be used based on the `useJsonp` property.
+     */
+    public $mimeType = null;
```

Comment:
@samdark I don't like the inconsistency with the `XmlResponseFormatter` but the `contentType` is incorrect and misleading since we do not allow to set a custom `Content-Type` but just a MIME type instead.